src/Pim/Component/Catalog/Model/AbstractProductUniqueData.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Pim\Component\Catalog\Model;
+
+/**
+ * @author    Damien Carcel (damien.carcel@akeneo.com)
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AbstractProductUniqueData implements ProductUniqueDataInterface
```

Comment:
Why "Abstract" as it's not an abstract class?
It's weird to me to have this class having ALL the logic, then 2 class extending it for nothing (no custom/extra logic).
So as the `PublishedProductUniqueData` is exactly the same as the `ProductUniqueData`, why not use `ProductUniqueData` for the EE?